mixin/signals/overview.libsonnet

```diff
@@ -0,0 +1,304 @@
+function(this) {
+  local legendCustomTemplate = '{{instance}}',
+  local aggregationLabels = std.join(',', ['job', 'instance']),
+  filteringSelector: this.filteringSelector,
+  groupLabels: this.groupLabels,
+  instanceLabels: this.instanceLabels,
+  enableLokiLogs: false,  // Snowflake does not gather logs
+  legendCustomTemplate: legendCustomTemplate,
+  aggLevel: 'none',
+  aggFunction: 'avg',
+  alertsInterval: '5m',
+  discoveryMetric: {
+    prometheus: 'snowflake_table_active_bytes',
+  },
+  signals: {
+    warehouseActivityExecutedQueries: {
+      name: 'Warehouse activity executed queries',
+      nameShort: 'Warehouse activity executed queries',
+      description: 'The number of executed queries for the selected warehouse.',
+      type: 'raw',
+      unit: 'percentunit',
+      sources: {
+        prometheus: {
+          expr: 'snowflake_warehouse_executed_queries{%(queriesSelector)s, name=~"$warehouse"}',
```

Comment:
Where does the queriesSelector variable come from? Is there an edge case when it doesn't get populated?